spec/javascripts/Common/components/SearchInputWithSubmitButton.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import { mount } from 'enzyme'
+import { act } from 'react-dom/test-utils'
+import { Popover, SearchInput } from '@patternfly/react-core'
+
+import { SearchInputWithSubmitButton } from 'Common/components/SearchInputWithSubmitButton'
+import { mockLocation } from 'utilities/test-utils'
+
+import type { Props } from 'Common/components/SearchInputWithSubmitButton'
+
+const defaultProps = {
+  placeholder: ''
+}
+
+const mountWrapper = (props: Partial<Props> = {}) => mount(<SearchInputWithSubmitButton {...{ ...defaultProps, ...props }} />)
+
+it('should render itself', () => {
+  const wrapper = mountWrapper()
+  expect(wrapper.exists()).toEqual(true)
+})
+
+describe('Popover is or is not visible', () => {
+  const event = {} as React.FormEvent<HTMLInputElement>
+  const wrapper = mountWrapper()
```

Comment:
The wrapper should be mounted in every test, otherwise your unir tests will depend on each other.